packages/client/lib/rpc/types.ts

```diff
@@ -0,0 +1,8 @@
+export interface RpcCallTx {
+  from?: string
+  to: string
```

Comment:
Can this also be `null`? Or is this the empty string in RPC? (If we want to create a contract)

Reply:
well this is for `eth_call` which is not meant for contract creation, in the [wiki docs](https://eth.wiki/json-rpc/API#eth_call) `to` is required.

for `eth_estimateCall` it could be optional though, so maybe i'll rename the type to `RpcTx` and `to` can be optional. I have already written a `requiredField` validator for `eth_call` that is enforced as middleware 👍

Reply:
fixed in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1130/commits/487d85ee67b7d400ee8f4720160afdd1d5bdc813